packages/react-toast/src/components/index.tsx

```diff
@@ -1,34 +1,58 @@
 export { default as Toast } from "./Toast";
 import React from "react";
-import styled from "styled-components";
+import styled, { createGlobalStyle, ThemeProvider } from "styled-components";
+import toastCss from "react-toastify/dist/ReactToastify.css";
+import { Brand } from "@trycourier/react-provider";
+import { Theme } from "../types";
 
 import Body from "./Body";
 import { toastStyles } from "./Toast/styled";
 import { ICourierToastMessage } from "./Toast/types";
 
 const Styled = styled.div(toastStyles);
+const GlobalStyle = createGlobalStyle`${toastCss}`;
+
+export const ToastBody: React.FunctionComponent<
+  ICourierToastMessage & {
+    theme: Theme;
+    brand: Brand;
+  }
+> = ({ theme, brand, ...props }) => {
+  props.icon = props.icon ?? brand?.inapp?.icons?.message;
```

Comment:
Could help me understand what configs or settings comes from studio design vs directly from the component props and what is the place to make more additions if I were to get more things coming from Studio designer?

Reply:
props.icon means that the icon is coming from a websocket which has higher prescedent than brand icon